econet_base.yaml

```diff
@@ -13,6 +13,7 @@ substitutions:
   external_components_source: github://esphome-econet/esphome-econet@${github_ref}
   logger_level: WARN
   econet_update_interval: 30s
+  econet_throttle_interval: ${econet_update_interval}
```

Comment:
I don't quite like this. Whoever wants to throttle they can just increase `econet_update_interval`. 30s is already infrequent enough. And whoever wants to add throttling can extend the sensors in their config and add whatever filters they want. Having the existing `delta` filter for `accuracy_decimals: 0` makes sense to me since by default users don't see the decimal points for sensors that high precision isn't that important.